packages/engine-test-utils/src/__tests__/engine-server/enginev2.spec.ts

```diff
@@ -189,3 +185,65 @@ describe("engine, config/", () => {
     });
   });
 });
+
+describe("engine, init", () => {
+  test("handles broken frontmatter gracefully", async () => {
```

Comment:
instead of writing this directly here, can you add it to ENGINE_PRESETS under INIT? this way, it will test both the engine response and the engine API response in `engine.spec.ts`

Reply:
There are actually tests already in `INIT` for notes with broken frontmatters and broken schemas. I deleted this test, and extended the existing ones to check that the initialization completed despite the caught error.

Reply:
And thanks for the suggestion, there was actually a bug that occurred with the API response but not engine!

Reply:
awesome. then our tests are doing their jobs :)